src/main/java/seedu/socket/ui/HelpWindow.java

```diff
@@ -51,7 +51,7 @@ public class HelpWindow extends UiPart<Stage> {
             + "9. Viewing a person's detail: view\n"
             + "Format: view INDEX\n\n";
     public static final String HELP_MESSAGE_PROJECT = "\n1. Adding a project: addpj\n"
-            + "Format: addpj [n/NAME] [h/REPO HOST] [r/REPO NAME] [d/DEADLINE] [m/MEETING]\n\n"
+            + "Format: addpj n/PROJECT NAME h/REPO HOST r/REPO NAME d/DEADLINE [m/MEETING]\n\n"
```

Comment:
Good spot, but would it better if it follows the UG format, PROJECT_NAME, REPO_NAME and etc